src/DotLiquid/SyntaxCompatibilityEnum.cs

```diff
@@ -24,5 +24,15 @@ public enum SyntaxCompatibility
         /// Behavior as of DotLiquid 2.2a
         /// </summary>
         DotLiquid22a = 221,
+
+        /// <summary>
+        /// Behavior as of DotLiquid 2.2b
+        /// </summary>
+        DotLiquid22b = 222,

```

Comment:
Current version of DotLiquid is 2.3. I would support just calling it DotLiquid24 = 240 and before merging bumping the version flag on the build system.

Reply:
I'll change this to DotLiquid24 (which will be the same as the one on the render branch).

Reply:
Done.